chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/component/ChaosMonkeySchedulerTest.java

```diff
@@ -86,8 +84,4 @@ public void shouldTriggerRuntimeScopeRunAttack() {
         return cronTask -> cronTask.getExpression().equals(schedule);
     }
 
-    @Before
-    public void initMocks() {
-        MockitoAnnotations.initMocks(this);
```

Comment:
Since this class is extended with `MockitoExtension` the call is superfluous.
(I suspect it wasn't even need before.)